plugins/Toolbox/src/Toolbox.py

```diff
@@ -735,8 +674,10 @@ def _onDownloadComplete(self, file_path: str) -> None:
             self.openLicenseDialog(package_info["package_id"], license_content, file_path)
             return
 
-        self.install(file_path)
-        self.subscribe(package_info["package_id"])
+        package_id = self.install(file_path)
+        if package_id != package_info["package_id"]:
+            Logger.error("Installed package {} does not match {}".format(package_id, package_info["package_id"]))
```

Comment:
... and then what? :-)

Reply:
I suggest ignoring this as a fluke, but i'll add it to the CCB. I think this check was added by me, so an error in the log is an improvement over silent failure

Reply:
> an error in the log is an improvement over silent failure

Agreed!